src/pages/posthighlight.tsx

```diff
@@ -0,0 +1,61 @@
+import { useContext } from "react";
+import { FaChevronLeft } from "react-icons/fa";
+import OpenSaucedLogo from "../assets/opensauced-logo.svg";
+import { RouteContext } from "../App";
+import GhOpenGraphImg from "../content-scripts/components/githhub-open-graph";
+
+
+const PostOnHighlight = () => {
+  const { setCurrentPage } = useContext(RouteContext);
+
+  return (
+    <div className="grid grid-cols-1 divide-y divider-y-center-2 min-w-[320px]">
+        <header className="flex justify-between">
+          <div className="flex items-center gap-2">
+            <button
+              className="rounded-full p-2 bg-slate-700 hover:bg-slate-700/50"
+              onClick={() => {
+                setCurrentPage("home");
+              }}
+            >
+              <FaChevronLeft className="text-osOrange text-white" />
+            </button>
+
+            <img
+              alt="OpenSauced logo"
+              className="w-[100%]"
+              src={OpenSaucedLogo}
+            />
+          </div>
+        </header>
+
+        <main className="text-white">
+
+        <GhOpenGraphImg
+          githubLink={"https://github.com/open-sauced/insights/pull/1184"}
```

Comment:
@Anush008 @diivi how can I use the current link on the tab? I tried document.location.href and window.location.href but it did not work here. Is there anything I need to check?

Reply:
https://stackoverflow.com/questions/18436245/how-to-fetch-url-of-current-tab-in-my-chrome-extension-using-javascript maybe this will help?